test/functional/data.spec.js

```diff
@@ -0,0 +1,49 @@
+const got = require('got');
+const {expect} = require('chai');
+
+const data = require('../../data.json');
+
+function checkURL(url, head = false) {
+	const method = head ? 'head' : 'get';
+	return got[method](url, {throwHttpErrors: false}).then(({statusCode}) => {
+		if (!head && statusCode === 403) {
+			return checkURL(url, true);
+		}
+
+		expect(statusCode).to.equal(200);
+	});
+}
+
+describe('swag-for-dev', function () {
+	it('data.json is valid', function () {
```

Comment:
I thing you should keep the require in the test loop, either by adding it to `data.json is valid` or creating a first `it` case that loads it to data? So if we use reporters in the future it won't crash and send invalid output?

Reply:
If the tests fail, data shouldn't be sent to external reporters, right? Especially in PRs 🤔

Reply:
My bad, your're right